manifests/v1alpha2/katib/db/deployment.yaml

```diff
@@ -0,0 +1,52 @@
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: katib-db
+  namespace: kubeflow
+  labels:
+    app: katib
+    component: db
+spec:
+  replicas: 1
+  template:
+    metadata:
+      name: katib-db
+      labels:
+        app: katib
+        component: db
+    spec:
+      containers:
+      - name: katib-db
+        image: mysql:8.0.3
+        args:
+        - --datadir
+        - /var/lib/mysql/datadir
+        env:
+          - name: MYSQL_ROOT_PASSWORD
+            valueFrom:
+              secretKeyRef:
+                name: katib-db-secrets
+                key: MYSQL_ROOT_PASSWORD
+          - name: MYSQL_ALLOW_EMPTY_PASSWORD
+            value: "true"
+          - name: MYSQL_DATABASE
+            value: "katib"
+        ports:
+        - name: dbapi
+          containerPort: 3306
+        readinessProbe:
+          exec:
+            command:
+            - "/bin/bash"
+            - "-c"
+            - "mysql -D $$MYSQL_DATABASE -p$$MYSQL_ROOT_PASSWORD -e 'SELECT 1'"
+          initialDelaySeconds: 5
+          periodSeconds: 2
+          timeoutSeconds: 1
+        volumeMounts:
+        - name: katib-mysql
+          mountPath: /var/lib/mysql
+      volumes:
+      - name: katib-mysql
+        persistentVolumeClaim:
+          claimName: katib-mysql
```

Comment:
Will we provide the PV config for users?

Reply:
I think, we have to provide it, like in v1alpha1. 
Some users don't have storage classes in their k8s clusters to bound PVC.

Reply:
If some users do not have storage classes, then how can we create PV for the users? Do you mean that we use the default storage class to do it?

Reply:
If users don't have StorageClass they need to follow this instruction: https://github.com/kubeflow/katib#katib.
They need to create PV manually from the yaml file.

Reply:
Ah! I missed pv/pvc.

Reply:
Yeah, I agree. Thus we could add a default config for the PV and PVC.

Reply:
in fact, files in manifest are used to test. In kubeflow product, only PV will not be created in ksonnet/kustimize

Reply:
We can create README file under kustomize Katib manifest, how to create PV, like we have here: https://github.com/kubeflow/kubeflow/tree/master/kubeflow/katib#katib-1